atomic_reactor/plugins/post_import_image.py

```diff
@@ -80,12 +80,14 @@ def run(self):
 
         self.osbs = get_openshift_session(self.workflow, self.openshift_fallback)
         self.get_or_create_imagestream()
-        self.process_tags()
+
         try:
-            self.osbs.import_image(self.imagestream_name, tags=self.get_trackable_tags())
+            self.osbs.import_image_tags(self.imagestream_name, self.get_trackable_tags(),
+                                        self.docker_image_repo, insecure=self.insecure_registry)
         except TypeError:
-            self.log.info('Falling back to calling import_image without tags')
-            self.osbs.import_image(self.imagestream_name)
+            self.log.info('Falling back to calling import_image instead of import_image_tags')
+            self.process_tags()
+            self.osbs.import_image(self.imagestream_name, tags=self.get_trackable_tags())
```

Comment:
Coveralls reporting is missing for some reason, but I don't think we're testing this whole block.